utilities/decrypt_data/decrypt.py

```
@@ -0,0 +1,90 @@
+        conn = psycopg2.connect(
+            host=CONFIG["db"]["host"],
+            port=CONFIG["db"]["port"],
+            dbname=CONFIG["db"]["dbname"],
+            user=CONFIG["db"]["user"],
+            password=CONFIG["db"]["password"]
+        )
+        cur = conn.cursor()
+
+        # Build SELECT query for all columns + id
+        select_columns = ", ".join([CONFIG["id_column"]] + CONFIG["columns"])
+        cur.execute(f"SELECT {select_columns} FROM {CONFIG['table']}")
```

Ruff (0.11.9)

Possible SQL injection vector through string-based query construction

(S608)